Klipper_Files/Extra module/ercf.py

```diff
@@ -959,6 +962,13 @@ def _load_to_end_of_bowden(self, length):
             return
 
     def _home_to_extruder(self, length, step):
+        if not self.toolhead_use_stallguard:
+            return self._home_to_extruder(length, step)
```

Comment:
FATAL:
This line produces a stack overflow... it should be call home_to_extruder_without_stallguard!

Reply:
Good spot! I refactored some things to remove duplication and missed some bits!